dsbridge/src/main/java/wendu/dsbridge/HuluActivity.java

```diff
@@ -0,0 +1,27 @@
+package wendu.dsbridge;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+/**
+ * Created by allensun on 1/24/18.
+ * on Tubitv.com, allengotstuff@gmail.com
+ */
+public class HuluActivity extends AppCompatActivity {
+
+    private DWebView dWebView;
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_hulu_play);
+
+        dWebView = (DWebView)findViewById(R.id.hulu_webview);
+        dWebView.setBackgroundColor(Color.BLACK);
+
+        dWebView.clearCache(true);
+        dWebView.loadUrl("put your webview url in there");
```

Comment:
insert your web url there, for security reason, i didn't put it on commit @springuper